styles.css

```diff
@@ -0,0 +1,114 @@
+html{
+    background-color: #f5eeee;
+    font-size: 16px;
+}
+
+
+
+h1{
+    color: #ff5566;
+    background-color:darkslategray;
+    text-align: center;
+    font-size: 3rem;
+    padding: 1rem;
+    font-family: "Impact", sans-serif;
+    margin: 0;
+    
+}
+
+nav{
+    background-color: darkslategray;
+    position: sticky;

```

Comment:
sticky ist eine Anweisung die leider nicht in allen Browsern funktioniert. Vorsichtig damit ;)